### PR TITLE
polymer 3.x (per import)

### PR DESCRIPTION
**Pull request by pixelass**
# Importable version

Allows this repository to be imported directly into the browser.

* `global` has been ported to es6 and is now hosted under `<polymer-redux>/helpers`
  * The required features should be basic enough to make this plausible
* `PolymerRedux` can be imported for polymer 3.x and assigned to a variable instead of the global scope
* entirely removes bower

The following import will now work as expected (without a build process)

## Use the import

```js
import PolymerRedux from '../node_modules/polymer-redux/src/index.js'
```

## No more `<script>` tags and global pollution

```html
<script src="/node_modules/polymer-redux/lib/index.js"></script>
```

### CODE REVIEW

File: helpers/README.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Helpers
```
Comment (tur-nr) [inline helpers/README.md]: Can we move this to `./src`.

File: src/index.js [hunk 2]
```diff
@@ -1,8 +1,7 @@
-import window from 'global/window';
-import console from 'global/console';
-
+import {window, console} from '../helpers/global/index.js';
+import {get} from '../node_modules/@polymer/polymer/lib/utils/path.js'
```
Comment (tur-nr) [inline src/index.js]: Is there a reason why we aren't just importing `@polymer/polymer/lib/utils/path.js`. Rollup/Webpack will resolve this. Is this a browser module specific import?

File: src/index.js [hunk 3]
```diff
@@ -187,6 +188,17 @@ export default function PolymerRedux(store) {
 					});
 					return originalAction(...args);
 				};
+				// Copy props from the original action to the proxy.
+				// see https://github.com/tur-nr/polymer-redux/issues/98
+				Object.keys(originalAction).forEach(prop => {
+					action[prop] = originalAction[prop];
+				});
+
+				// Copy props from the original action to the proxy.
```
Comment (tur-nr) [inline src/index.js]: We seem to a have a merge conflict, this is duplicated.

**Review by tur-nr — CHANGES_REQUESTED** [review]
Thanks for the contribution, just a few changes. Thanks!

File: src/index.js (hunk 2 above)
Comment (pixelass) [reply src/index.js]: I am still testing this. I haven't seen any other packages with 3rd party dependencies, therefore I will do some testing. 

It is in fact a browser import. If this is packed via rollup we would load polymer(`Path.get`) twice in our projects.

Lets just keep this open for a while. I might have a cleaner option.

File: src/index.js (hunk 3 above)
Comment (pixelass) [reply src/index.js]: I noticed. thx for the hint

**Comment by pixelass** [comment]
I am currently having problems adjusting the tests since bower is still required by web-components-tester. https://github.com/Polymer/web-component-tester/issues/582

(.. getting there though)

So it seems like we can't fully remove bower yet 😢

**Comment by tur-nr** [comment]
See [polymer-3](https://github.com/tur-nr/polymer-redux/tree/polymer-3) branch.

**Comment by TimPietrusky** [comment]
I successfully included [polymer-3](https://github.com/tur-nr/polymer-redux/tree/polymer-3) into my project and everything is working. 

Thank you super much!

**Pull request closed without merging**